src/tests/JIT/Regression/JitBlue/Runtime_100437/Runtime_100437.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
```

Comment:
```suggestion
```
This test should work both with optimizations on and off.

Reply:
Fixed by commit cd1ebe0